ast/edit/testcase_test.go

```diff
@@ -74,3 +81,134 @@ myVar = 4`
 		t.Errorf("unexpected package count: want: 0, got: %d", len(transformed))
 	}
 }
+
+func TestTestcaseTransformImport(t *testing.T) {
+	fs := &inmemFS{
+		files: map[string]string{
+			"a/a_test.flux": `package a_test
+import "testing/assert"
+want = 4
+testcase a {
+	assert.equal(want: want, got: 2 + 2)
+}
+`,
+			"b/b_test.flux": `package b_test
+import "testing/assert"
+testcase b {
+	import "../a/a_test"
```

Comment:
What is the reasoning for relative import paths? How will this work in the context of IDPE importing a Flux test? Its not clear to me where on the filesystem a Flux test lives relative to an IDPE test.

Reply:
The idea that I had was we didn't really have a concept of where the top level was. Tests just live in arbitrary places so we'd have to reference them relative to where they are.

I'm not sure how we'll consolidate the test cases from OSS and other sources. I had not thought of that. I'll try to have a better answer tomorrow.

Reply:
I'd appreciate some feedback on ways to consolidate the sources, but I thought it might be better to go more into detail about why I figured relative imports were better in this circumstance.

The reason why I figured relative imports were better were for relatively minor reasons, but enough that I figured it was bumpy.

1. The semantics of importing a file vs a package were different. Even if flux only supports one file packages at the moment, the stated semantics are that you import a path and it includes all of the non-test flux files in that directory as part of a single package. This mean that importing a single file didn't make much sense to me and seemed confusing.
2. In light of the above, it was unclear to me if the syntax should be to import `"strings"`, `"strings_test"`, or `"strings/strings_test"`. The first seemed to match the most with what import meant, but it also implied that you would want to import the entire package of test files. The second made no sense because while it referenced the package in what I viewed as a correct way, it also implied every test file and it didn't even point to a filesystem. The third was what was proposed, but it means that this specific import would reference a specific file rather than a package which seemed to confuse the meaning of import. If I were doing this from scratch, my preference would be for test files to have the same package as the main package and to import the main package and have it include the test files. But, I also figured this probably would have issues with scoping since every test uses `inData` as a global variable and those would conflict.
3. It seemed to me that since user-defined imports didn't exist, it would give the wrong impression if I added the concept of an import path. If we had a concept of an import path, then I would have used that to find the files.
4. Building on the lack of an import path, the test runner runs files that it finds and not packages. There's no current method to determine what the root directory is so when you are running the tests for a package, there's no way to know what the root directory is to descend from there. Go used GOPATH and now it uses `go.mod` as a way of defining what the root is. We don't have those so relative paths are really all I had without introducing these concepts.

With the above, it was my opinion that the above concepts were too much for this project and it would torpedo the project. This was a judgment and I think it might be good to evaluate that judgment. I think that might be helpful for when we want to expose this to external users to ask ourselves what we would want to change and make sure that we put a priority on the requirements for that project before we begin working on it.

I considered moving away from using `import` because I thought the semantics might be too confusing, but it was my opinion that adding a new keyword to the language would be worse. Thinking about it, another possibility is to go back to the old syntax and do something like:

```
testcase a with "b" {
}
```

The advantage of this is it would distinguish the importing mechanism completely from testcase so there wouldn't be any confusion about the mechanics. I still think we'd have to do relative paths though since we don't have a concept of a root directory.